test/studies/lcals/LCALSMain.chpl

```diff
@@ -0,0 +1,1203 @@
+    var loop_names = suite_run_info.loop_names;
+    var len_id = new vector(string);
+
+    len_id.resize(suite_run_info.loop_length_names.size());
+    for ilen in 0..#len_id.size() {
+      len_id[ilen] = suite_run_info.loop_length_names[ilen].substring(1);
+    }
+
+
+    const equal_line = "===========================================================================================================\n";
+    const dash_line = "------------------------------------------------------------------------------------------------------------\n";
+    const dash_line_part = "-------------------------------------------------------\n";
+    const dot_line_part = "............................................\n";
+
+    var ver_info = buildVersionInfo();
+    outchannel.write("\n\n\n");
+    outchannel.write(equal_line);
+    outchannel.write(equal_line);
+    outchannel.writeln("LCALS compilation summary: ", ver_info);
+
+    outchannel.write("\n\n");
+    outchannel.write(equal_line);
+    outchannel.write(equal_line);
+
+    var max_name_len = 0;
+    for iloop in 0..#loop_names.size() {
+      max_name_len = max(max_name_len, loop_names[iloop].length);
+    }
+    var max_var_name_len = 0;
+    for ilv in 0..#nvariants {
+      max_var_name_len = max(max_var_name_len, run_loop_variants[ilv].length);
+    }
+
+    var var_field = "Variant(length id)";
+    var var_field_len = max(var_field.length, max reduce [ilv in 0..#nvariants]  run_loop_variants[ilv].length + 3);
+    var prec = 32;
+    var prec_buf = prec + 8;
+
+    outchannel.writeln("Loop name -->");
+    outchannel.writef("%-*s", var_field_len+1, var_field);
+    outchannel.writef("%*s", prec_buf, "Check Sum    ");
+    outchannel.writef("%-*s\n", prec_buf, "        Delta from reference");
+    outchannel.write(dash_line);
+
+
+    for iloop in 0..#loop_names.size() {
+      var ref_variant_stat = suite_run_info.getLoopStats(run_loop_variants[0])[iloop];
+      var ref_chksum = new vector(real);
+      ref_chksum.resize(ref_variant_stat.loop_chksum.size());
+      for i in 0..#ref_chksum.size() {
+        ref_chksum[i] = ref_variant_stat.loop_chksum[i];
+      }
+      if loop_names[iloop].length != 0 && ref_variant_stat.loop_is_run {
+        if iloop > 1 then
+          outchannel.write("\n", dash_line_part);
+        outchannel.write(loop_names[iloop], "(", iloop, ") --> ");
+        for ilv in 0..#nvariants {
+          var stat = suite_run_info.getLoopStats(run_loop_variants[ilv])[iloop];
+          if stat.loop_is_run {
+
+            if ilv == 0 then outchannel.writeln();
+            else outchannel.write(dot_line_part);
+
+            for ilen in 0..#stat.loop_length.size() {
+              if stat.loop_run_count[ilen] > 0 {
+                var var_string = run_loop_variants[ilv] + "(" + len_id[ilen] + ")";
+                outchannel.writef("%-*s", var_field_len+1, var_string);
+                //outchannel.writef("%*s", prec_buf, stat.loop_chksum[ilen]);
```

Comment:
Is this and the other commented out stuff below meant to stick around, or was it for short term debugging only?